pkgs/desktops/gnome-3/devtools/gnome-builder/default.nix

```diff
@@ -0,0 +1,78 @@
+{stdenv
+, clang
+, defaultIconTheme
+, devhelp
+, fetchgit
+, gettext
+, gnome_themes_standard
+, gspell
+, gtk3
+, gtksourceview
+, hicolor_icon_theme
+, json_glib
+, jsonrpc_glib
+, libdazzle
+, libgit2-glib
+, libpeas
+, libxml2
+, llvmPackages
+, meson
+, ninja
+, pcre
+, pkgconfig
+, pkgs
```

Comment:
Why is this here?

Reply:
Leftover from some tinkering. Will be removed with next push.